.github/workflows/upstream-nightly.yml

```diff
@@ -73,12 +64,7 @@ jobs:
           echo "JAX_ENABLE_X64=$JAX_ENABLE_X64"
           echo "JAX_ENABLE_CHECKS=$JAX_ENABLE_CHECKS"
           echo "JAX_SKIP_SLOW_TESTS=$JAX_SKIP_SLOW_TESTS"
-          pytest -n auto --tb=short -rf --maxfail=20 \
-              --report-log output-${{ matrix.python-version }}-log.jsonl \
-              tests \
-              || (
-                echo 'ARTIFACTS_AVAILABLE=true' >> $GITHUB_OUTPUT && false
-              )
+          pytest -n auto --tb=short --maxfail=20 tests examples
```

Comment:
AFAICT We weren't previously testing `examples` here. Seems reasonable that we want to - just double checking that this was intentional!

Reply:
I added it intentionally – I think it will be useful to have that extra test coverage